test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs

```diff
@@ -128,6 +128,11 @@ public void TempDirectoryCleanup()
 
         if (!string.IsNullOrWhiteSpace(inIsolation))
         {
+            if (inIsolation != "/InIsolation")
+            {
+                // TODO: The whole inIsolation should be just a bool, but it is not, and it's changing in other PR.
```

Comment:
Probably enum rather than bool, we should have an option, for forcing in process and asserting on it, in isolation, and letting the framework figure it out.